src/vision/projects/MonsterMesh.md

```diff
@@ -15,8 +15,10 @@ This is a sample project for use within the various ["status quo"] or ["shiny fu
 
 * **What makes embedded projects like MonsterMesh different from others?**
     * Embedded developers need to write error-free applications outside of the comfort zone of an operating system. Rust helps to prevent many classes of programming errors at compile time which inspires confidence in the software quality and and cuts time intensive build-flash-test iterations.
-    * Embedded developers needs good hardware abstraction. Frameworks in other languages do not provide the sophistacted memory mapped IO to safe type abstraction tooling which have been created by the Rust teams.
+    * Embedded developers needs good hardware abstraction. Frameworks in other languages do not provide the sophisticated memory mapped IO to safe type abstraction tooling which have been created by the Rust teams.
     * Embedded developers care about hard real time capabilities; the concept of "you only pay for what you use" is very important in embedded applications. The combination of the inherently asynchronous interrupt handling of microcontrollers with the Rust async building blocks are a perfect match to effortlessly create applications with hard realtime capabilities.
-    * Embedded developers are particularly appreciative of strong tooling support. The availibity of the full environment via `rustup` and the integration of the full toolchain with `cargo` and `build.rs` make her very happy because she can focus on what she does best instead of having regular fights with the environment.
+    * Embedded developers are particularly appreciative of strong tooling support. The availability of the full environment via `rustup` and the integration of the full toolchain with `cargo` and `build.rs` make her very happy because she can focus on what she does best instead of having regular fights with the environment.
 * **Does MonsterMesh require a custom tailored runtime?**
-    * Yes! The tradeoffs for an embedded application like MonsterMesh and a typical server are very different. Further, most server-grade frameworks are not `#[no_std]` compatible and far exceeded the available footprint on the sensor nodes.
\ No newline at end of file
+    * Yes! The tradeoffs for an embedded application like MonsterMesh and a typical server are very different. Further, most server-grade frameworks are not `#[no_std]` compatible and far exceeded the available footprint on the sensor nodes.
```

Comment:
Hmm, pre-existing, but I feel like this is misleading. MonsterMesh may not need a runtime tailored to *it specifically* but tailored to its *needs*.